docs/en/docs/advanced/sql-databases-sqlalchemy.md

```diff
@@ -0,0 +1,87 @@
+# Async Databases with SQLAlchemy
+
+This tutorial will cover the usage of SQLAlchemy async parts as an ORM.
+
+!!! tip
+    We strongly suggest going through the non async tutorial first ([SQL (Relational) Databases](../tutorial/sql-databases.md){.internal-link target=_blank}), since this tutorial will only cover the async parts of SQLAlchemy.
+
+!!! warning
+    SQLAlchemy asyncio support is currently in beta and is not stable.
```

Comment:
SQLAlchemy 1.4 is currently in beta, however the asyncio support is currently in alpha as noted here:

https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#change-3414

Reply:
Don't see a difference between 'beta' and 'alpha' personally